README.md

```diff
@@ -4,27 +4,17 @@
     </a>
 </p>
 
-
-<div align="center">
-<a rel="nofollow">
-    <img src="docs/img/python-badge.svg" alt="Python" />
-</a>
-<a rel="nofollow">
-    <img src="docs/img/pypi-badge.svg" alt="Pypi" />
-</a>
-<a rel="nofollow">
-    <img src="docs/img/license-badge.svg" alt="License" />
-</a>
-<a rel="nofollow">
-    <img src="docs/img/cov-badge.svg" alt="Coverage" />
-</a>
-<a rel="nofollow">
-    <img src="docs/img/mypy-badge.svg" alt="Styling" />
-</a>
-<a rel="nofollow">
-    <img src="docs/img/cov-style.svg" alt="Styling" />
-</a>
-</div>
+[![Contributions Welcome](https://img.shields.io/badge/contributions-welcome-brightgreen.svg)](CONTRIBUTING.md)
+[![Python Versions](https://img.shields.io/pypi/pyversions/jumanji.svg?style=flat-square)](https://badge.fury.io/py/jumanji)
```

Comment:
Is sending the user to the PyPI project what we want for the python versions badge? Would linking to the [python doc](https://www.python.org/doc/versions/) not make more sense here?